doc/nrf/ug_nrf91_features.rst

```diff
@@ -249,61 +269,152 @@ Sensor and GNSS data are sent to the cloud only during the data transfer phase.
+P-GPS is a form of assistance, where the device can download up to two weeks of predicted satellite Ephemerides data.
+Normally, devices connect to the cellular network approximately every two hours for up-to-date satellite Ephemeris information or they download the Ephemeris data from the acquired satellites.
+P-GPS enables devices to determine the exact orbital location of the satellite without connecting to the network every two hours with a trade-off of reduced accuracy of the calculated position over time.
+Note that P-GPS requires more memory compared to regular A-GPS.
 
-* :ref:`http_full_modem_update_sample` sample - performs a full firmware OTA update of the modem.
-* :ref:`http_modem_delta_update_sample` sample - performs a delta OTA update of the modem firmware.
-* :ref:`http_application_update_sample` sample - performs a basic application FOTA update.
-* :ref:`aws_fota_sample` sample - performs a FOTA update via MQTT and HTTP, where the firmware download is triggered through an AWS IoT job.
-* :ref:azure_fota_sample sample - performs a FOTA update from the Azure IoT Hub
-* :ref:`asset_tracker_v2` application - performs FOTA updates of the application, modem (delta), and boot (if enabled).
+Also, note that due to satellite clock inaccuracies, not all functional satellites will have Ephemerides data valid for two weeks in the downloaded PGPS package.
+This means that the number of satellites having valid predicted Ephemerides reduces in number roughly after ten days.
+Hence, the GNSS module needs to download the Ephemeris data from the satellite broadcast if no predicted Ephemeris is found for that satellite to be able to use the satellite.
+
+SUPL compared to nRF Cloud as location service
+----------------------------------------------
+
+SUPL library uses approximately 100 kB of flash memory, which takes up a big chunk of the flash memory.
+nRF Cloud as location services instead has been designed to use minimal memory space on the device and send minimal possible data over the air to save power.
+
+Low Accuracy Mode
+-----------------
+
+Low accuracy mode allows the GNSS receiver to accept a looser criterion for a fix with four or more satellites or by using a reference altitude to allow for a fix using only three satellites.
+This has a tradeoff of reduced accuracy.
+This reference altitude can be from a recent valid normal fix or it can be artificially injected.
+See the section on low accuracy mode in :ref:`nrfxlib:gnss_interface` for more information about low accuracy mode and its usage.
+
+Samples using GNSS in |NCS|
+===========================
+
+There are many examples in |NCS| that use GNSS.
+Following is a list of the samples and applications with some information about the GNSS usage:
+
+* :ref:`asset_tracker_v2` application - Uses nRF Cloud for A-GPS or P-GPS  or a combination of both. The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.
```

Comment:
Asset Tracker v1 also supports it.  It is not been removed from NCS so we should mention it here.
```
* :ref:`asset_tracker`

Reply:
https://github.com/nrfconnect/sdk-nrf/commit/f2fc904119de6f39b2d86d21c94fc73eb492b986
inline with ncs 1.7.0 asset tracker v1 is deprecated and therefore I avoid linking it here